### DISCUSSION TITLE
Volume rebalance option

### DISCUSSION THREAD

**Discussion by hamidreza-hosseini** [seaweedfs/seaweedfs#2649] [Ideas]
Hi @chrislusf 

This is the output of "volume.balance" command from weed shell:

```
> volume.balance --help
Usage of volume.balance:
  -collection string
        collection name, or use "ALL_COLLECTIONS" across collections, "EACH_COLLECTION" for each collection (default "EACH_COLLECTION")
  -dataCenter string
        only apply the balancing for this dataCenter
  -force
        apply the balancing plan.
```

It would be great if there were any options for rebalancing from a server based on max possible volumes count not the whole cluster...
for example, I have a disk with 7.3TB space and there are 128 volumes in it (the possible and correct count is 73), if there was an option that weed understand that the possible Max count is 73 and 128 is 55 more than the possible count and it starts the rebalancing from the source (It can be a server )

Would it be possible to consider this option for weed?

**Comment by chrislusf** [comment]
Maybe. Welcome to learn some Go code and create a PR.
